### ISSUE TITLE
Identation lines get copied/pasted

### ISSUE THREAD

**Issue by lmiq** [Yggdroot/indentLine#357]
The identation lines get copied with the code. For example:

```julia
function f(x)
  if x == 1
  │ println(x)
  end
end
```

Of course, this invalidates the code when pasting from one file to the other, or to a REPL. Is there a workaround around this? (I am using Mint 20.1 with the default terminal, copy/paste, etc)

**Comment by honeyglazed** [comment]
Have you tried yanking the lines to the system clipboard instead `"+y` - assuming your vim build supports clipboard.

**Comment by lmiq** [comment]
Yes, there are workarounds like that. But it is annoying that one cannot use "select -> middle click"  to copy/paste. But I was told that changing that would be complicated. Unfortunately that makes me not using identation in vim most of the time.

**Comment by LemonadeJoseph** [comment]
You could maybe map a shortcut to the command `:IndentLinesToggle` and toggle off the indent symbols before copying?

Then after you're done copying, run the last command (either though history or with `@:`) to toggle them back on
That's how I do it...
